doc/quickstart/actinia_quickstart.rst

```diff
@@ -714,11 +318,11 @@ commands from above can be executed in the following way:
    acp r.info elev@test_mapset
    acp r.info slope_elev@test_mapset
 
-Super admin tasks
```

Comment:
Can we remove the super admin tasks as well? Just to keep the quickstart nice and tight?

Reply:
Yes, ok for me